core/src/main/java/org/elasticsearch/common/rounding/Rounding.java

```diff
@@ -61,6 +65,14 @@ public final long round(long value) {
      */
     public abstract long nextRoundingValue(long value);
 
+    public abstract XContentBuilder doXContentBody(XContentBuilder builder, Params builderParams) throws IOException;
```

Comment:
We don't, this is left over from an idea I had for parsing and rendering Roundings. I have now removed this